src/main/scala/cognite/spark/v1/RelationConfig.scala

```diff
@@ -22,7 +22,7 @@ final case class RelationConfig(
     subtrees: AssetSubtreeOption,
     ignoreNullFields: Boolean,
     rawEnsureParent: Boolean,
-    enableSinglePartitionDeleteAssetHierarchy: Boolean // flag to test whether single partition helps avoid NPE in asset hierarchy builder
+    useSinglePartition: Boolean // a flag for helping to test NPEs by moving the data frame to a single partition
```

Comment:
```suggestion
    useSinglePartitionForAssetHierarchyDeletion: Boolean // a flag for helping to test NPEs by moving the data frame to a single partition
```
or something. Maybe invert the name so that default values is false and it'd mean do repartition to 1. Also let's use repartition as default here